$OpenBSD: patch-openconnect-internal_h,v 1.1 2012/01/16 22:58:54 sthen Exp $

Fix memory leak of CSTP deflated packets, and resend current pkt on reconnect
b39d34 upstream

--- openconnect-internal.h.orig	Fri Nov  4 21:06:13 2011
+++ openconnect-internal.h	Thu Jan 12 09:39:59 2012
@@ -146,6 +146,7 @@ struct openconnect_info {
 	int owe_ssl_dpd_response;
 	struct pkt *deflate_pkt;
 	struct pkt *current_ssl_pkt;
+	struct pkt *pending_deflated_pkt;
 
 	z_stream inflate_strm;
 	uint32_t inflate_adler32;
